applications/matter_weather_station/README.rst

```diff
@@ -175,6 +183,40 @@ The ``build_thingy53_nrf5340_cpuapp`` parameter specifies the output directory f
 
       File not found: ./ncs/nrf/applications/matter_weather_station/configuration/thingy53_nrf5340_cpuapp/prj_shell.conf
 
+Pre-prepared factory data support
```

Comment:
```suggestion
Enabling factory data support
```

Reply:
I would like to somehow highlight that this section shows how to flash example factory data provided by us. Enabling factory data support is also part of it, but user may not want to use our data, but generate own ones (Generating new factory data section).

Reply:
Yes, but you are providing specific command about how to do something. That implies a specifically formatted header. There is barely any contextual info here, so usage of the header you suggest is not possible. Unless we provide more information about what pre-prepared factory data support is, but this is done earlier.

Reply:
Rephrased to Programming pre-prepared factory data and removed info about enabling factory data. More details sent on private channel.